packages/assets-controllers/src/TokenListController.test.ts

```diff
@@ -1211,7 +1126,11 @@ describe('TokenListController', () => {
         resolve();
       });
 
-      network.setRpcTarget('http://localhost', '56');
+      controllerMessenger.publish('NetworkController:providerConfigChange', {
```

Comment:
Great to see more examples of how to use the top-level controller messenger in tests to simulate events. Not enough examples of this in our tests, I was asked about how to do this a few times now.